src/modules/core/selectors/transactions.ts

```diff
@@ -51,18 +51,26 @@ export const transactionByHash = (state: RootStateRecord, hash: string) =>
 export const groupedTransactions = createSelector(
   allTransactions,
   transactions =>
-    transactions // Create groups of transations which have 'em
-      .groupBy(tx => tx.group && tx.group.id) // Convert groups to lists and sort by no in group
-      .map(txGroup => txGroup.toList().sortBy(tx => tx.group && tx.group.index)) // Merge the ungrouped transactions into the ordered map. It's important that all iterators here have the same type (OrderedMap)
-      // For proper typing we create single value arrays for all of the single transactions
-      // We're using key.toString() here to not confuse flow. The output of allTransactions always has a string id in group
+    transactions
+      // Create groups of transations which have 'em
```

Comment:
I adjusted the formatting of this a little bit (I guess prettier did weird things to it at one point).